install.sh

```diff
@@ -137,7 +139,8 @@ if [ "${BACKEND_GDB}" -eq 1 ]; then
     GDB_PYTHON=$(${GDB} -batch -q --nx -ex 'pi import sys; print(sys.executable)')
     GDB_PYTHON="${GDB_PYTHON/%$GDB_PYVER/}${GDB_PYVER}"
 
-    install_packages
+    install_packages $PYVER
```

Comment:
This will throw a missing `python-dev` package on Debian bullseye and newer.

Keeping only next line L#143 is enough